node/service/src/tests.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(dead_code)]
```

Comment:
maybe create an issue / todo so this doesnt live forever in this file by accident

Reply:
@drahnr

Reply:
Done: https://github.com/paritytech/polkadot/issues/3674